src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -493,22 +482,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 _argument = argument
             End Sub
 
-            Public Overrides ReadOnly Property ArgumentKind As ArgumentKind
-                Get
-                    ' Do the VB bound trees encode named arguments?
-                    Return ArgumentKind.Positional
-                End Get
-            End Property
-
             Public Overrides ReadOnly Property InConversion As IOperation
                 Get
-                    Return _argument.InConversion
+                    Dim conversion As BoundExpression = _argument.InConversion
+                    ' The bound trees can contain a simple placeholder as a conversion. Don't treat this as a conversion.
+                    If conversion IsNot Nothing AndAlso conversion.Kind <> BoundKind.ByRefArgumentPlaceholder Then
```

Comment:
No--in this case there is no actual conversion. My guess is that inserting the placeholders as "conversions" triggers generating code to do the copying in and out.